integration/spark-common-test/src/test/scala/org/apache/carbondata/spark/testsuite/sortcolumns/TestSortColumns.scala

```diff
@@ -33,16 +33,15 @@ class TestSortColumns extends QueryTest with BeforeAndAfterAll {
       CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT, "dd-MM-yyyy")
 
     SparkUtil4Test.createTaskMockUp(sqlContext)
-    dropTable
     CarbonProperties.getInstance()
       .addProperty(CarbonCommonConstants.CARBON_TIMESTAMP_FORMAT, "dd-MM-yyyy")
+    dropTestTables
```

Comment:
not required

Reply:
I add common method QueryTest for all test case in org.apache.spark.sql.test.util.QueryTest, so subclass should change the method name